### DISCUSSION TITLE
UPDATE README.md

### DISCUSSION THREAD

**Discussion by C0FF33P4G3** [AntonOsika/gpt-engineer#591] [Ideas]
I recently tried using it by following the steps in the README.md file and it does not work, please update the file.

I keep getting this error when i try to export/set the API key 

openai.error.AuthenticationError: No API key provided. You can set your API key in code using 'openai.api_key = <API-KEY>', or you can set the environment variable OPENAI_API_KEY=<API-KEY>). If your API key is stored 
in a file, you can point the openai module at it with 'openai.api_key_path = <PATH>'. You can generate API keys in the OpenAI web interface. See https://platform.openai.com/account/api-keys for details.

**Comment by webd-com** [comment]
I had the same problem, and the way I solved it is by using this: $env:OPENAI_API_KEY="[your api key]" on powershell 
My mistake was that I was using "set" on Powershell  when it was meant for cmd

**Comment by C0FF33P4G3** [comment]
Ohhhhh I appreciate your email back I get you thank you very much I will
fix it.
